frontend/app/views/joiner/thankyou.scala.html

```diff
@@ -51,7 +51,12 @@ <h2 class="page-section__headline">@title</h2>
 }
 
 @main(title) {
-
+<script type="text/javascript">
+   window.productData = {
```

Comment:
Maybe add this to window.guardian.productData instead? That would be more consistent with subs.